data.json

```diff
@@ -0,0 +1 @@
+{"id": 123, "name": "James"}
```

Comment:
File to remove ?

Reply:
Hmm, I don't remember adding that one. I'll remove it.

Reply:
This file is produced when I run `make docs`. This happens when running `make docs` from `master`. I'll delete the file in this PR, and raise an issue about it.

Reply:
Lodged as #1992

Reply:
Yep I already did the fix in https://github.com/samuelcolvin/pydantic/pull/1990 yesterday. Thanks for creating the issue I linked the fix with it